src/lib/utils/dateFnsUtils.ts

```diff
@@ -54,11 +54,17 @@ export function formatMilliseconds(milliseconds: number) {
     const minutes = Math.floor((totalSeconds % 3600) / 60)
     const seconds = totalSeconds % 60
 
-    const hoursString = hours === 0 ? '' : hours + ' timer'
-    const minutesString = minutes === 0 ? '' : minutes + ' minutter'
-    const secondsString = seconds === 0 ? '' : seconds + ' sekunder'
-
-    return `${hoursString} ${minutesString} ${secondsString}`
+    const hoursString =
+        hours === 0 ? '' : hours === 1 ? '1 time' : hours + ' timer'
+    const minutesString =
+        minutes === 0 && hours === 0
+            ? ''
+            : minutes === 1
+              ? '1 minutt'
+              : minutes + ' minutter'
+    const secondsString = seconds === 1 ? '1 sekund' : seconds + ' sekunder'
```

Comment:
Legge til sjekk for 0 som i minutes og hours også slik at det ikke blir 0 sekunder